src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_ARRAY_CAPACITY = 16;
+    private static final int RESIZE_NUMBER = 2;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_ARRAY_CAPACITY];
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the constructor.